### DISCUSSION TITLE
Issue with setting up my Development Environment

### DISCUSSION THREAD

**Discussion by Nehemiah60** [CiviWiki/OpenCiviWiki#1489] [Help and support]
I have encountered issues while setting up my dev environment. The first is when I try poetry install, the response I get is No dependencies to install or update. I skipped it and tried all the other steps, and everything worked. However, when running my project, the display tends to be so different from what is expected. So this is my project display. I don't understand, since there are no tabs when I click the user profile.
I will appreciate your support.
Thanks.
![Screenshot (411)](https://user-images.githubusercontent.com/87471613/215054585-0f1f1db9-56ab-44a5-a483-2e2b37871b7e.png)
![Screenshot (412)](https://user-images.githubusercontent.com/87471613/215054592-f4090d12-0d5e-4b50-a9f6-be7004b8cd7a.png)

**Comment by brylie** [comment]
It seems like Poetry has already installed the dependencies and Django is running from the above screenshot. Were you able to run `python manage.py runserver` without error?

**Reply by Nehemiah60** [reply]
Yead, I did run the server without an error. I can also register comfortably. However, the profile page seems a bit off. There is nothing at all. Like I can see the following tab, or followers tab. Or anything. It displays nothing. This is my screenshot.
![Screenshot (410)](https://user-images.githubusercontent.com/87471613/215075362-f8e2d959-2678-4ddf-8f00-f51ae1e5248d.png)

**Reply by brylie** [reply]
OK, that is expected, since the profile page is under development. Otherwise, your development environment seems fine.

**Reply by Nehemiah60** [reply]
Okay, I had identified a few open issues I would wish to work on. Let me get straight into it.
